README.md

```diff
@@ -58,3 +58,22 @@ command:
 ```
 firefox com.ibm.streamsx.metrics/doc/spldoc/html/index.html &
 ```
+
+### Building Project
+The MetricsSink operator requires you to build jars and include them in your project. You will also need to have an ElasticSearch server running.
```

Comment:
Please provide at least one sample application for your operator. For this sample, please also describe the required preparation, so that somebody can import and run the SPL sample.